src/main/java/net/sf/jabref/exporter/OOCalcDatabase.java

```diff
@@ -67,7 +66,7 @@ public OOCalcDatabase(BibtexDatabase bibtex, Set<String> keySet) {
             }
         }
 
-        entries = new SortedList(entryList, new FieldComparatorStack(comparators));
+        entries = new SortedList<BibtexEntry>(entryList, new FieldComparatorStack<BibtexEntry>(comparators));
```

Comment:
OK! Most of the things I tried to solve was based on warnings in Eclipse. Just to make it clear, I should have written

`entries = new SortedList(entryList, new FieldComparatorStack<BibtexEntry>(comparators));`

as entries was already defined as `SortedList<BibtexEntry>`? I might just have seen the warning, changed in SortedList, seen that it was still there and then actually read the warning...